locales/it-IT/general.json

```diff
@@ -110,7 +110,7 @@
     "rated": "Valutato",
     "recommended": "Non Consigliato",
     "report": "Report",
-    "reports": "Reports",
+    "reports": "Report",
```

Comment:
is this intended? the modification makes the plural appear singular?

Reply:
Yes because in italian the terms that are english also if they are plural are written in the singular form